llarp/router/rc_lookup_handler.cpp

```diff
@@ -235,6 +238,26 @@ namespace llarp
       LogError("we have no bootstrap nodes specified");
     }
 
+    if(useWhitelist)
+    {
+      std::set< RouterID > lookupRouters;
+      {
+        static constexpr size_t LookupPerTick = 25;
+        // if we are using a whitelist look up a few routers we don't have
+        util::Lock l(&_mutex);
+        for(const auto &r : whitelistRouters)
```

Comment:
consider starting at a random index rather than iterating, so that each time we do this lookup loop we're probably looking up a different set (we'll eventually get back to any we missed, so may as well not try them multiple times in a row)

Reply:
Should this also be looking up routers even if we don't have them in the nodedb to refresh the RCs (i.e. to keep them from going stale and expiring)?

Reply:
So that was supposed to be happening anyway, but I derped a bit when I refactored that bit out of Router.  It's fixed in this PR as well, see changes to rc_lookup_handler.cpp

Reply:
Randomized in #1041